src/Okta.Sdk/Internal/PkixCertPayloadHandler.cs

```diff
@@ -41,7 +41,17 @@ protected override HttpContent GetRequestHttpContent(HttpRequest httpRequest)
                 throw new ArgumentNullException("request payload");
             }
 
-            return new StringContent((string)httpRequest.Payload, Encoding.UTF8, ContentType);
+            var content = string.Empty;
+            if (httpRequest.Payload is string stringPayload)
+            {
+                content = stringPayload;
+            }
+            else if (httpRequest.Payload is byte[] bytePayload)
+            {
+                content = Convert.ToBase64String(bytePayload);
```

Comment:
Sanity-check question: Is this conversion necessary when you are passing a byte[]? Is this how the request looks in our docs?

Reply:
Yes, this payload handler handles content for the Content-Type "application/pkix-cert" and will always send the payload as a base64 encoded string.  So, if a byte array is provided, that byte array needs to be converted to base64 in order for the headers in the message to be valid.